project_report/tex/requirements.tex

```diff
@@ -9,7 +9,13 @@ \subsubsection{Functional \& Non-Functional Requirements}
 \subsubsection{The Physical Setup (The Brewery Machine)}
 
 \subsubsection{The Simulator}
-
+The group are going to use the simulator software to test the software in the
+development cycle.
+The simulator is not a replacement of the machine since there is only
```

Comment:
Start sentence with: "It is important to note that the simulator is not a......"